mistletoe/latex_renderer.py

```diff
@@ -23,7 +23,12 @@ def render_emphasis(self, token):
         return '\\textit{{{}}}'.format(self.render_inner(token))
 
     def render_inline_code(self, token):
-        return '\\verb|{}|'.format(self.render_inner(token))
+        content = ""
+        # there should be only one children...
+        for children in token.children:
+            content += self.render_raw_text(children, escape=False)
+
+        return '\\verb|{}|'.format(content)
```

Comment:
OK, but when talking about (not) escaping, we should also cope with `|` here? Please see https://tex.stackexchange.com/questions/10629/escape-in-verb: It looks like one can choose nearly any character as the delimiter, but the character must not be present within the rendered text.